website/docs/getting-started/features.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Features
+sidebar_position: 2
+---
+
+# Features provided by odo
+
+odo follows "create and push" workflow for almost everything. It means, as a user, when you "create" something the information (or manifest) is stored in a configuration file, and then upon doing a "push" it gets created  on the Kubernetes cluster.
+
+One can take an existing git repository and create an odo component from it, which can be pushed to a Kubernetes cluster.
+
+odo helps deploy and link multiple components and services with each other. By using odo, application developers can develop, test, debug and deploy microservices based applications on Kubernetes without having a deep understanding of the platform.
+
+### What can odo do?
+
+Full details of what each odo command is capable of doing can be found in the "Command Reference" sections.
+Below is a summary of its most important capabilities:
+* Create a manifest to deploy applications on Kubernetes cluster; odo creates the manifest for existing projects as well as new ones.
```

Comment:
Before this, you can explain that odo uses generic Docker images to run the app in the cluster: no need to write Dockerfiles

Reply:
s/a manifest/Kubernetes manifests/
at least the first occurrence

Reply:
> Before this, you can explain that odo uses generic Docker images to run the app in the cluster: no need to write Dockerfiles

I am not sure if you were asking to do it the way I have done. PTAL.

> s/a manifest/Kubernetes manifests/ 

Actually, I was referring to the devfile that odo creates. Didn't want to use the "d" word so put it as manifest. It's the devfile library that eventually creates things, so I wasn't thinking of "Kubernetes manifests" while writing this. WDYT?